lib/src/dynamics/island.dart

```diff
@@ -174,34 +149,34 @@ class Island {
     _listener = listener;
 
     if (_bodies == null || _bodyCapacity > _bodies.length) {
-      _bodies = new List<Body>(_bodyCapacity);
+      _bodies = List<Body>(_bodyCapacity);
     }
     if (_joints == null || _jointCapacity > _joints.length) {
-      _joints = new List<Joint>(_jointCapacity);
+      _joints = List<Joint>(_jointCapacity);
     }
     if (_contacts == null || _contactCapacity > _contacts.length) {
-      _contacts = new List<Contact>(_contactCapacity);
+      _contacts = List<Contact>(_contactCapacity);
     }
 
     // dynamic array
     if (_velocities == null || _bodyCapacity > _velocities.length) {
       final List<Velocity> old =
-          _velocities == null ? new List<Velocity>(0) : _velocities;
-      _velocities = new List<Velocity>(_bodyCapacity);
+          _velocities == null ? List<Velocity>(0) : _velocities;
```

Comment:
Can't we use the `??` operator here?

```dart
final List<Velocity> old = _velocities ?? List<Velocity>(0)
```

Reply:
Done, I'll see if there are more null checks that can be replaced.